teamsite/main/templates/main/home.html

```diff
@@ -1,24 +1,306 @@
-<!doctype html>
-<html lang="en">
+{% load static %}
+<!DOCTYPE html>
+<html>
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Home</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>online course website</title>
+
+	<!---custom css link---->
+	<link rel="stylesheet" href="{% static 'main/css/final.css' %}">
+	<link rel="stylesheet" href="https://unpkg.com/boxicons@latest/css/boxicons.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700;800;900&display=swap" rel="stylesheet">
 </head>
 <body>
-<ul>
-    <li>
-        User: {{ user }}
-    </li>
-    <li>
-        {% if user.is_authenticated %}
-            <a href="{% url 'logout' %}">Logout</a>
-        {% else %}
-            <a href="{% url 'login' %}">Login</a>
-        {% endif %}
-    </li>
-</ul>
+
+	<!---start header---->
+	<header>
+		<a href="#" class="logo">
```

Comment:
is there a sense to have href="#" at all here?

Reply:
Yes,it makes sense.When you scroll down ,for example,to footer and click on logo you will smoothly move from bottom to header.